packages/angular/projects/dev/src/main.ts

```diff
@@ -5,13 +5,13 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app/app.module';
 import { environment } from './environments/environment';
-import { runCssVarsPolyfill } from '@clr/core';
+// import { runCssVarsPolyfill } from '@clr/core';
```

Comment:
Actually, one little thing ... are we saving this for something?

Reply:
I wanted to talk to @coryrylan about this

Reply:
This is fine to leave commented or remove, we will start using the demo apps for core testing instead of the dev app